Adding vendors page (#131)
apache/iceberg-docs: landing-page/content/common/vendors.md

---
title: "Vendors"
url: vendors
disableSidebar: true
---
<!--
 - Licensed to the Apache Software Foundation (ASF) under one or more
 - contributor license agreements.  See the NOTICE file distributed with
 - this work for additional information regarding copyright ownership.
 - The ASF licenses this file to You under the Apache License, Version 2.0
 - (the "License"); you may not use this file except in compliance with
 - the License.  You may obtain a copy of the License at
 -
 -   http://www.apache.org/licenses/LICENSE-2.0
 -
 - Unless required by applicable law or agreed to in writing, software
 - distributed under the License is distributed on an "AS IS" BASIS,
 - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 - See the License for the specific language governing permissions and
 - limitations under the License.
 -->

> ## Vendors shipping Apache Iceberg

Reviewer: nit: "Vendors With Iceberg Table Support"? or maybe "Vendors Supporting Iceberg Tables"?

This page contains some of the vendors who are shipping and supporting Apache Iceberg in their products

> ### [Cloudera](http://clouera.com)

Reviewer 2: ```suggestion
### [Cloudera](http://cloudera.com)
```

> Cloudera Data Platform integrates Apache Iceberg to the following components:

Reviewer: I like the content here! Later on we can put this in a shortcode that will add some custom styling and also make this more like a form to keep every new addition consistent. That way it's easier to just fill out name, link, description, feature-list, code-snippet, etc with character limits built into the styling.

* Apache Hive, Apache Impala, and Apache Spark to query Apache Iceberg tables
* Cloudera Data Warehouse service providing access to Apache Iceberg tables through Apache Hive and Apache Impala
* Cloudera Data Engineering service providing access to Apache Iceberg tables through Apache Spark
* The CDP Shared Data Experience (SDX) provides compliance and self-service data access for Apache Iceberg tables
* Hive metastore, which plays a lightweight role in providing the Iceberg Catalog
* Data Visualization to visualize data stored in Apache Iceberg

https://docs.cloudera.com/cdp/latest/cdp-iceberg/topics/iceberg-in-cdp.html

> 

Reviewer 2: ```suggestion

### [Starburst](http://starburst.io)

Starburst is a commercial offering for the [Trino query engine](https://trino.io). Trino is a distributed MPP SQL query engine that can query data in Iceberg at interactive speeds. Trino also enable you to join Iceberg tables with an [array of other systems](https://trino.io/docs/current/connector.html). With Starburst, we offer both an [enterprise deployment](https://www.starburst.io/platform/starburst-enterprise/) and a [fully managed service](https://www.starburst.io/platform/starburst-galaxy/) to make managing and scaling Trino a flawless experience. We also provide customer support and house many of the original contributors to the open-source project that know Trino best. Learn more about [our Iceberg connecter](https://docs.starburst.io/latest/connector/iceberg.html).
```

---
title: "Vendors"
url: vendors
disableSidebar: true
---
<!--
 - Licensed to the Apache Software Foundation (ASF) under one or more
 - contributor license agreements.  See the NOTICE file distributed with
 - this work for additional information regarding copyright ownership.
 - The ASF licenses this file to You under the Apache License, Version 2.0
 - (the "License"); you may not use this file except in compliance with
 - the License.  You may obtain a copy of the License at
 -
 -   http://www.apache.org/licenses/LICENSE-2.0
 -
 - Unless required by applicable law or agreed to in writing, software
 - distributed under the License is distributed on an "AS IS" BASIS,
 - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 - See the License for the specific language governing permissions and
 - limitations under the License.
 -->

> ## Vendors Supporting Iceberg Tables

Reviewer 3: @jasonhughes248, would you like to add a section for Dremio?

Reviewer 4: @rdblue yeah, that'd be great, thanks!

doesn't look like I have the permissions necessary to add it myself, so here's the content:

### [Dremio](https://www.dremio.com/)

Dremio is the easy and open data lakehouse platform. With Dremio, an organization can easily build and manage a data lakehouse in which data is stored in open formats like Apache Iceberg and can be processed with Dremio’s interactive SQL query engine and non-Dremio processing engines. [Dremio Cloud](https://www.dremio.com/get-started/) provides these capabilities in a fully managed offering.

[Dremio Sonar](https://www.dremio.com/platform/sonar/) is a lakehouse query engine that provides interactive performance and DML on Apache Iceberg, as well as other formats and data sources. It gives data providers the ability to quickly provision new views and metrics on Apache Iceberg tables and other data sources without ETL pipelines. It gives data consumers governed self-service and the ability to interact with a logical data model.

[Dremio Arctic](https://www.dremio.com/platform/arctic/) is a lakehouse catalog and optimization service for Apache Iceberg. Arctic makes it easy to manage tables in the lakehouse, and automatically optimizes the data in the background to ensure high-performance access through Dremio Sonar, Spark, Flink, Snowflake, Athena, EMR, and other engines.

Reviewer 3: Added, but I cut out some of the sentences that would appear like the ASF project is making claims about Dremio (as we did for the other entries here). Feel free to suggest edits if you think I got it wrong!

Reviewer 4: ah @rdblue makes sense. yup your changes look good

This page contains some of the vendors who are shipping and supporting Apache Iceberg in their products

### [Cloudera](http://clouera.com)

Cloudera Data Platform integrates Apache Iceberg to the following components:
* Apache Hive, Apache Impala, and Apache Spark to query Apache Iceberg tables
* Cloudera Data Warehouse service providing access to Apache Iceberg tables through Apache Hive and Apache Impala
* Cloudera Data Engineering service providing access to Apache Iceberg tables through Apache Spark
* The CDP Shared Data Experience (SDX) provides compliance and self-service data access for Apache Iceberg tables
* Hive metastore, which plays a lightweight role in providing the Iceberg Catalog
* Data Visualization to visualize data stored in Apache Iceberg

https://docs.cloudera.com/cdp/latest/cdp-iceberg/topics/iceberg-in-cdp.html

### [Starburst](http://starburst.io)

> Starburst is a commercial offering for the [Trino query engine](https://trino.io). Trino is a distributed MPP SQL query engine that can query data in Iceberg at interactive speeds. Trino also enables you to join Iceberg tables with an [array of other systems](https://trino.io/docs/current/connector.html). With Starburst, we offer both an [enterprise deployment](https://www.starburst.io/platform/starburst-enterprise/) and a [fully managed service](https://www.starburst.io/platform/starburst-galaxy/) to make managing and scaling Trino a flawless experience. We also provide customer support and house many of the original contributors to the open-source project that know Trino best. Learn more about [our Iceberg connecter](https://docs.starburst.io/latest/connector/iceberg.html).

Reviewer 3: @bitsondatadev, could you remove personal pronouns from this? I think "Starburst offers ..." is more clear that "With Starburst, we offer ..." because it isn't clear who "we" is, and I don't want there to be confusion that the Iceberg community endorses a particular vendor.

> Starburst is a commercial offering for the [Trino query engine](https://trino.io). Trino is a distributed MPP SQL query engine that can query data in Iceberg at interactive speeds. Trino also enables you to join Iceberg tables with an [array of other systems](https://trino.io/docs/current/connector.html). With Starburst, we offer both an [enterprise deployment](https://www.starburst.io/platform/starburst-enterprise/) and a [fully managed service](https://www.starburst.io/platform/starburst-galaxy/) to make managing and scaling Trino a flawless experience. We also provide customer support and house many of the original contributors to the open-source project that know Trino best. Learn more about [our Iceberg connecter](https://docs.starburst.io/latest/connector/iceberg.html).

Reviewer 2: ```suggestion
Starburst is a commercial offering for the [Trino query engine](https://trino.io). Trino is a distributed MPP SQL query engine that can query data in Iceberg at interactive speeds. Trino also enables you to join Iceberg tables with an [array of other systems](https://trino.io/docs/current/connector.html). Starburst offers both an [enterprise deployment](https://www.starburst.io/platform/starburst-enterprise/) and a [fully managed service](https://www.starburst.io/platform/starburst-galaxy/) to make managing and scaling Trino a flawless experience. Starburst also provides customer support and house many of the original contributors to the open-source project that know Trino best. Learn more about [the Starburst Iceberg connecter](https://docs.starburst.io/latest/connector/iceberg.html).
```

 - distributed under the License is distributed on an "AS IS" BASIS,
 - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 - See the License for the specific language governing permissions and
 - limitations under the License.
 -->

## Vendors Supporting Iceberg Tables

This page contains some of the vendors who are shipping and supporting Apache Iceberg in their products

### [Cloudera](http://cloudera.com)

Cloudera Data Platform integrates Apache Iceberg to the following components:
* Apache Hive, Apache Impala, and Apache Spark to query Apache Iceberg tables
* Cloudera Data Warehouse service providing access to Apache Iceberg tables through Apache Hive and Apache Impala
* Cloudera Data Engineering service providing access to Apache Iceberg tables through Apache Spark
* The CDP Shared Data Experience (SDX) provides compliance and self-service data access for Apache Iceberg tables
* Hive metastore, which plays a lightweight role in providing the Iceberg Catalog
* Data Visualization to visualize data stored in Apache Iceberg

https://docs.cloudera.com/cdp/latest/cdp-iceberg/topics/iceberg-in-cdp.html

> ### [Starburst](http://starburst.io)

Reviewer 3: ```suggestion
### [Snowflake](http://snowflake.com/)

[Snowflake](https://www.snowflake.com/data-cloud/) is a single, cross-cloud platform that enables every organization to mobilize their data with Snowflake’s Data Cloud. Snowflake supports Apache Iceberg by offering [native support for Iceberg Tables](https://www.snowflake.com/blog/iceberg-tables-powering-open-standards-with-snowflake-innovations/) for full DML as well as connectors to [External Tables](https://www.snowflake.com/blog/expanding-the-data-cloud-with-apache-iceberg/) for read-only access.

### [Starburst](http://starburst.io)
```

> Starburst is a commercial offering for the [Trino query engine](https://trino.io). Trino is a distributed MPP SQL query engine that can query data in Iceberg at interactive speeds. Trino also enables you to join Iceberg tables with an [array of other systems](https://trino.io/docs/current/connector.html). Starburst offers both an [enterprise deployment](https://www.starburst.io/platform/starburst-enterprise/) and a [fully managed service](https://www.starburst.io/platform/starburst-galaxy/) to make managing and scaling Trino a flawless experience. Starburst also provides customer support and houses many of the original contributors to the open-source project that know Trino best. Learn more about [the Starburst Iceberg connecter](https://docs.starburst.io/latest/connector/iceberg.html).

Reviewer 4: ```suggestion
Starburst is a commercial offering for the [Trino query engine](https://trino.io). Trino is a distributed MPP SQL query engine that can query data in Iceberg at interactive speeds. Trino also enables you to join Iceberg tables with an [array of other systems](https://trino.io/docs/current/connector.html). Starburst offers both an [enterprise deployment](https://www.starburst.io/platform/starburst-enterprise/) and a [fully managed service](https://www.starburst.io/platform/starburst-galaxy/) to make managing and scaling Trino a flawless experience. Starburst also provides customer support and houses many of the original contributors to the open-source project that know Trino best. Learn more about [the Starburst Iceberg connector](https://docs.starburst.io/latest/connector/iceberg.html).
```

Reviewer 4: just a heads up, minor typo 🙂

 - distributed under the License is distributed on an "AS IS" BASIS,
 - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 - See the License for the specific language governing permissions and
 - limitations under the License.
 -->

## Vendors Supporting Iceberg Tables

This page contains some of the vendors who are shipping and supporting Apache Iceberg in their products

### [Cloudera](http://cloudera.com)

Cloudera Data Platform integrates Apache Iceberg to the following components:
* Apache Hive, Apache Impala, and Apache Spark to query Apache Iceberg tables
* Cloudera Data Warehouse service providing access to Apache Iceberg tables through Apache Hive and Apache Impala
* Cloudera Data Engineering service providing access to Apache Iceberg tables through Apache Spark
* The CDP Shared Data Experience (SDX) provides compliance and self-service data access for Apache Iceberg tables
* Hive metastore, which plays a lightweight role in providing the Iceberg Catalog
* Data Visualization to visualize data stored in Apache Iceberg

https://docs.cloudera.com/cdp/latest/cdp-iceberg/topics/iceberg-in-cdp.html

> 

Reviewer 3: ```suggestion
### [Dremio](https://www.dremio.com/)

With Dremio, an organization can easily build and manage a data lakehouse in which data is stored in open formats like Apache Iceberg and can be processed with Dremio’s interactive SQL query engine and non-Dremio processing engines. [Dremio Cloud](https://www.dremio.com/get-started/) provides these capabilities in a fully managed offering.

* [Dremio Sonar](https://www.dremio.com/platform/sonar/) is a lakehouse query engine that provides interactive performance and DML on Apache Iceberg, as well as other formats and data sources.
* [Dremio Arctic](https://www.dremio.com/platform/arctic/) is a lakehouse catalog and optimization service for Apache Iceberg. Arctic automatically optimizes tables in the background to ensure high-performance access for any engine. Arctic also simplifies experimentation, data engineering, and data governance by providing Git concepts like branches and tags on Apache Iceberg tables.
```
